config_st.py

```diff
@@ -0,0 +1 @@
+Max_input=2
```

Comment:
constants go fully uppercased
```suggestion
MAX_INPUT=2
```

Reply:
done